MainModule/Server/Commands/Moderators.lua

```diff
@@ -892,6 +892,34 @@ return function(Vargs, env)
 				end
 			end
 		};
+		
+		RemoveLayeredClothings = {
+			Prefix = Settings.Prefix;
+			Commands = {"removelayeredclothings"};
```

Comment:
pls make short subcommand :)

Reply:
huh?

Reply:
It means that a command with a shorter name should exist because typing the full name is cumbersome